### PR TITLE
helmsman 3.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> go build -o /usr/local/Cellar/helmsman/3.0.0/bin/helmsman
build .: cannot find module for path .
```

**Comment by chenrui333** [comment]
Relates to #47627.

**Comment by chenrui333** [comment]
Hold this PR for a little bit until confirming with the upstream repo.

File: Formula/traefik.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def install
 
     cd "src/github.com/containous/traefik" do
       system "go", "generate"
-      system "go", "build", "-o", bin/"traefik", "./cmd/traefik"
+      system "go", "build", "-trimpath", "-o", bin/"traefik", "./cmd/traefik"
```
Comment (bayandin) [inline Formula/traefik.rb]: I think these changes shouldn't be in this commit

Comment (chenrui333) [reply Formula/traefik.rb]: Sure, I will remove it.

Comment (chenrui333) [reply Formula/traefik.rb]: Not quite sure that change got in, maybe doing two things at one time 😅

**Pull request closed without merging**